packages/eslint-config/package.json

```
@@ -0,0 +1,15 @@
+{
+  "name": "eslint-config-custom",
+  "private": true,
+  "version": "0.0.0",
+  "main": "eslint-config.js",
+  "dependencies": {
+    "@pagopa/eslint-config": "^3.0.0",
+    "@typescript-eslint/eslint-plugin": "^5.59.0",
+    "@typescript-eslint/parser": "^5.59.0",
+    "eslint": "^8.50.0",
```

```suggestion
    "eslint": "^8.51.0",
```
I'd bump to the latest version